.pylintrc

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
Unncessary file, imo.

Reply:
Not really. The rest of the checks except the manifest ones are correct for the Python code.

Reply:
Right ! Most of check are useless out of an odoo module context, but some are usefull, like sql-injection.